### DISCUSSION TITLE
Duplicate value

### DISCUSSION THREAD

**Discussion by Akaki-1991** [YetiForceCompany/YetiForceCRM#16706] [General] [answered]
Hi, can I make any fields in a manually created module have unique values? For example: as it is VAT in accounts module?

**Comment by vovpff** [comment]
I think what VAT is checked by some handler before saving. You can check how it works and write your own handler for your needs

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Duplicate this handler and set your fields
https://github.com/YetiForceCompany/YetiForceCRM/blob/7ee300ca64f3c541bf9a81665d2b214788907965/modules/Accounts/handlers/DuplicateVatId.php#L18

**Reply by Akaki-1991** [reply]
> Duplicate this handler and set your fields
> 
> https://github.com/YetiForceCompany/YetiForceCRM/blob/7ee300ca64f3c541bf9a81665d2b214788907965/modules/Accounts/handlers/DuplicateVatId.php#L18

can you help me, in which diractory i will save edited file? custom module?
